launcher/agent/agent_test.go

```diff
@@ -88,16 +87,9 @@ func TestAttest(t *testing.T) {
 		containerSignaturesFetcher signaturediscovery.Fetcher
 	}{
 		{
-			name:                       "all experiment flags disabled",
-			launchSpec:                 spec.LaunchSpec{},
-			principalIDTokenFetcher:    placeholderPrincipalFetcher,
-			containerSignaturesFetcher: signaturediscovery.NewFakeClient(),
-		},
-		{
-			name: "enable signed container",
+			name: "all experiment flags disabled",
```

Comment:
maybe we rename this now that theyre gone?
"Happy path" or something

Reply:
Done.